fil_actor_interface/src/macros.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019-2022 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+#[macro_export]
+macro_rules! parse_pending_transactions {
```

Comment:
It'd be nice to have some documentation here.

Reply:
added